eng/common/pipelines/templates/steps/verify-links.yml

```
@@ -5,6 +5,7 @@ parameters:
 steps:
 - task: PowerShell@2
   displayName: Link verification check
+  condition: ne(variables['Skip.VerifyLinks'], 'true')
```

```suggestion
  condition: and(succeeded(), ne(variables['Skip.VerifyLinks'], 'true'))
```

We have to add in the succeeded condition as well otherwise this will run on failures and canceled builds as well.